Snowflake.Data.Tests/UnitTests/ChunkDeserializerTest.cs

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2021 Snowflake Computing Inc. All rights reserved.
  */
 
-namespace Snowflake.Data.Tests
+namespace Snowflake.Data.Tests.UnitTests
```

Comment:
did you figure out how to run tests form given root folder only from command line?

Reply:
Yes - it is in the readme.